pkg/bpf/bpf_linux.go

```diff
@@ -98,12 +104,15 @@ func UpdateElement(fd int, key, value unsafe.Pointer, flags uint64) error {
 		flags: uint64(flags),
 	}
 
+	startTime := time.Now()
 	ret, _, err := unix.Syscall(
 		unix.SYS_BPF,
 		BPF_MAP_UPDATE_ELEM,
 		uintptr(unsafe.Pointer(&uba)),
 		unsafe.Sizeof(uba),
 	)
+	callDuration := time.Since(startTime)
```

Comment:
I would not use time.Now() and Time.Since() due the monotonic issues that we had seen. I would use spanStat that has a fix for that. 

(We already had problems in the endpoint metrics about this)

Reply:
Good point